doc/userguide/rules/flow-keywords.rst

```diff
@@ -305,4 +305,80 @@ Signature example::
 
  alert tcp any any -> any any (msg:"Flow longer than one hour"; flow.age:>3600; flowbits: isnotset, onehourflow; flowbits: onehourflow, name; sid:1; rev:1;)
 
-In this example, we combine `flow.age` and `flowbits` to get an alert on the first packet after the flow's age is older than one hour.
\ No newline at end of file
+In this example, we combine `flow.age` and `flowbits` to get an alert on the first packet after the flow's age is older than one hour.
+
+flow.pkts_toclient
+------------------
+
+Flow number of packets to client (integer)
+
+Syntax::
+
+ flow.pkts_toclient: [op]<number>
+
+The number of packets can be matched exactly, or compared using the _op_ setting::
+
+ flow.pkts_toclient:3    # exactly 3
+ flow.pkts_toclient:<3   # smaller than 3
+ flow.pkts_toclient:>=2  # greater or equal than 2
```

Comment:
>  But in the SV test, the flow that matches actually has 13 packets

Yes, the signature matches on the 10th packet of the 13-packet flow, not at the end of the flow.

The interesting part about this keyword is that you do not have to wait for the flow to be logged to act on it ;-)

Is it clearer ? How would you phrase it ?

Reply:
What you've added helps, maybe what we could do is add an explanation with a rule example, indicating that if the packet in question is, say, the tenth packet, the rule will be triggered, regardless of the flow having more packets. 

Still thinking about the test: the rule triggers only once, in case of exact match, but could trigger more than once, for a flow with 13 packets to the server, for instance, with a rule matching on `>=10` packets?